### ISSUE TITLE
Bug - UI issues / axe violations raised by sideNavLayout 

### ISSUE THREAD

**Issue by jenny-s51** [patternfly/patternfly-org#3622]
**Describe the problem**
A clear and concise description of the problem. Which pages are affected?
These axe violations of sideNavLayout were discovered upon review of #3573 - sideNavLayout may need structural updates to resolve these violations following conversation with @jessiehuff 

<img width="743" alt="Screenshot 2023-06-22 at 4 44 05 PM" src="https://github.com/patternfly/patternfly-org/assets/32821331/c8d69569-8fa5-4131-848f-b994e066b13c">

As noted by @thatblindgeye : 

> The toolbar items that include the theme toggle => release selector get kinda bunched up at smaller viewports (main issue is probably the release toggle getting cut off so much)
![image](https://github.com/patternfly/patternfly-org/assets/32821331/754b3108-7593-4ace-9e7e-1c2ab1f49ad1)
